### ISSUE TITLE
Show the window in the multitack view

### ISSUE THREAD

**Issue by megatux** [luizaugustomm/tomato#46]
Probably is hidden on purpose, but I find annoying that I can not drag the app window between workspaces.

**Comment by cassidyjames** [comment]
This looks like it's because the main window is a dialog, which is discouraged by GTK. elementary OS does not show dialogs in the Multitasking View.

**Comment by megatux** [comment]
Interesting. Maybe related to the merged branch called _tomato-as-dialog_.

**Comment by cassidyjames** [comment]
Correct. Instead of a dialog, Tomato should use the `.flat` style class on the decoration to style it similarly to a dialog. This is what several small apps, including [Dippi](https://github.com/csasidyjames/dippi) and [Palette](https://github.com/csasidyjames/palette) do.

**Comment by marcelobalexandre** [comment]
I created a draft PR (https://github.com/tomatoers/tomato/pull/64) to change the `MainWindow` to be a `Gtk.Window` instead of a `Gtk.Dialog`.

I'm not fluent on Vala so I'm not sure about which is the best approach to do this, please feel free to comment in my draft PR.

**Comment by marcelobalexandre** [comment]
The PR (https://github.com/tomatoers/tomato/pull/64) is ready for a review. ;)
